Frameworks/UIKit/UIButton.mm

```diff
@@ -262,9 +262,13 @@ - (void)initAccessibility {
 */
 - (void)setImage:(UIImage*)image forState:(UIControlState)state {
     _states[state].image = image;
-    WUXMImageBrush* imageBrush = ConvertUIImageToWUXMImageBrush(image);
-    if (imageBrush) {
-        _states[state].inspectableImage = [imageBrush comObj];
+    if (image) {
```

Comment:
>   if (image) { [](start = 1, length = 15)

nit: worth adding the same comment here as you did in the other two portions of the fix?